tests/test_core.py

```diff
@@ -34,9 +34,9 @@
 def wasm_simd(f):
   def decorated(self):
     if os.getenv('EMTEST_NO_D8'):
-      unittest.skip('Skipped because EMTEST_NO_D8 is set')
+      self.skipTest('Skipped because EMTEST_NO_D8 is set')
     if not config.V8_ENGINE or config.V8_ENGINE not in config.JS_ENGINES:
-      self.skipTest('wasm simd only supported in d8 for now')
+      self.fail('wasm simd only supported in d8 for now')
```

Comment:
Can you change this to `wasm simd only supported in d8 but d8 not found (set EMTEST_NO_D8 to skip these tests)`